relay-filter/src/browser_extensions.rs

```diff
@@ -189,6 +206,20 @@ mod tests {
         get_event_with_exception(ex)
     }
 
+    fn get_event_with_exception_value_and_os_name(val: &str, os_name: &str) -> Event {
+        let event = get_event_with_exception_value(val);
+
+        let tags = vec![Annotated::new(TagEntry(
+            Annotated::new("os.name".to_string()),
+            Annotated::new(os_name.to_string()),
+        ))];
+
+        Event {
+            tags: Annotated::new(Tags(PairList::from(tags))),
+            ..event
+        }
```

Comment:
I think this is more clear:
```suggestion
        let mut event = get_event_with_exception_value(val);
        event.tags = vec![Annotated::new(TagEntry(
            Annotated::new("os.name".to_string()),
            Annotated::new(os_name.to_string()),
        ))];
        event
```